app/models/idea.rb

```diff
@@ -1,2 +1,3 @@
 class Idea < ApplicationRecord
+	belongs_to :user
```

Comment:
Indentation looks a bit strange, is it two spaces or two tabs?
Try to change your settings in Sublime https://stackoverflow.com/a/22704309

Reply:
It's fixed